app/utils/group-by.js

```diff
@@ -0,0 +1,27 @@
+//
+// Code from https://github.com/HeroicEric/ember-group-by (MIT licensed)
+//
+import { A } from '@ember/array';
+import { get } from '@ember/object';
+import { isPresent } from '@ember/utils';
+
+export default function groupBy (collection, property) {
+  let groups = A();
+  let items = collection;
+
+  if (items) {
```

Comment:
maybe
if(!items) { 
return groups;
}

Reply:
Did you see that you can now suggest changes to commit in code comments? It's the first icon button in the comment topbar. Makes it much easier to see what the suggested change is as well, because the code behind the comment isn't cut off then...

Reply:
I think a single return is a bit cleaner here.